src/System.Windows.Forms/src/System/Windows/Forms/ToolStripManager.cs

```diff
@@ -54,14 +52,14 @@ internal static Font DefaultFont
     {
         get
         {
-            Font sysFont = null;
+            Font? sysFont = null;
 
             // We need to cache the default fonts for the different DPIs.
             if (DpiHelper.IsPerMonitorV2Awareness)
             {
                 int dpi = CurrentDpi;
 
-                Font retFont = null;
+                Font? retFont = null;
                 if (s_defaultFontCache.TryGetValue(dpi, out retFont) == false || retFont is null)
```

Comment:
if `s_defaultFontCache` can't hold null `Font`s, this can simply be
``
```suggestion
                if (!s_defaultFontCache.TryGetValue(dpi, out Font? retFont))
```